server/command/rotation.go

```diff
@@ -3,60 +3,20 @@
 
 package command
 
-import (
-	"github.com/pkg/errors"
-	"github.com/spf13/pflag"
-)
-
-func (c *Command) rotation(parameters []string) (string, error) {
-	subcommands := map[string]func([]string) (string, error){
-		commandAutopilot:   c.autopilotRotation,
-		commandAdd:         c.addRotation,
-		commandArchive:     c.archiveRotation,
-		commandDebugDelete: c.debugDeleteRotation,
-		commandForecast:    c.forecastRotation,
-		commandGuess:       c.guessRotation,
-		commandJoin:        c.joinRotation,
-		commandLeave:       c.leaveRotation,
-		commandList:        c.listRotations,
-		commandNeed:        c.rotationNeed,
-		commandShow:        c.showRotation,
-		commandUpdate:      c.updateRotation,
+import "github.com/mattermost/mattermost-plugin-solar-lottery/server/utils/md"
+
+func (c *Command) rotation(parameters []string) (md.MD, error) {
+	subcommands := map[string]func([]string) (md.MD, error){
```

Comment:
Not sure I understand this comment, elaborate please?

Reply:
In `command.go` we have this function:
```
func (c *Command) commands() map[string]func([]string) (md.MD, error)
```
This function is at some point used as:
```
c.handleCommand(c.commands(), parameters)
```
Here is a similar pattern, but we declare the map inside the function, instead of creating a new function like commands() to create the map.

Probably for these cases with subcommands is enough as it is, but I am wondering if there would be any improvement from having a `rotationSubcommands()` function (and similar for any other command with subcommands).

Reply:
Ah! https://github.com/levb/mattermost-plugin-solar-lottery/blob/lev-task-refactor/server/command/command_test.go#L107 uses it to `runCommand`, so it needed to be shareable. Returning a copy - just so that no need to worry about tests stepping onto each other.